tests/acceptance/test_base.py

```diff
@@ -150,6 +150,17 @@ class TestMemoryCache:
         with pytest.raises(TypeError):
             SimpleMemoryCache(random_attr="wtf")
 
+    @pytest.mark.asyncio
+    async def test_set_cancel_previous_ttl_handle(self, memory_cache):
+        await memory_cache.set(pytest.KEY, "value", ttl=0.1)
+
+        await asyncio.sleep(0.05)
+        assert await memory_cache.get(pytest.KEY) == "value"
+        await memory_cache.set(pytest.KEY, "new_value", ttl=0.1)
+
+        await asyncio.sleep(0.05)
+        assert await memory_cache.get(pytest.KEY) == "new_value"
+
```

Comment:
Hey @minhtule lets move this test to `TestCache` and use the `cache` fixture. This way we ensure all backends behave in a consistent way :)

Reply:
That makes sense. I've moved the test there.